### ISSUE TITLE
www.animenova.org

### ISSUE THREAD

**Issue by sol666** [reek/anti-adblock-killer#4099]
It started recently that the website comment section suddenly stopped displaying captcha code. Ithought the webpage had some problems. But the problem continued the next day. I tried posting a comment and it gave an alert "cannot access comment post". I run multiple adblocks (if one does not recognize an ad the others might) I decided to turn the adblockers off and refresh the site. turns out the moment I turned of AdBlockPlus the captcha code appeared  and it allowed posting in the comment section. The next day it started doing the same thing again, this time ABP was off. So I started turning off all the adblockers one by one and it did not allow comment posting until the last one was off. I concluded the webpage is using some kind of ad detector that won't allow access to comment section unless all adblock apps are off. I tried whitelisting the site on ABP but it still refused access to comment posting until ABP was turned off. Not even the "anti-adblock killer" by reek seems to bypass this adblock detection based access. It felt like some kind of new ad-block detection based security code so I am posting it here maybe someone can figure it out.

**Comment by uBlock-user** [comment]
https://github.com/reek/anti-adblock-killer/wiki/AAK-Dead-and-Discontinued

**Comment by KiLL4FuNK** [comment]
use @sol666

uBlock Origin Google Chrome

https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm?hl=es


uBlock Origin Mozilla Firefox

https://addons.mozilla.org/es/firefox/addon/ublock-origin/


uBlock Origin Opera

https://addons.opera.com/es-419/extensions/details/ublock/


uBlock Origin Microsoft Edge

https://www.microsoft.com/en-us/store/p/ublock-origin/9nblggh444l4

**Comment by sol666** [comment]
Yeah, I did, it doesn't work. It worked for the first few hours, then
suddenly the problem came back. As if whatever anti adblock is running
there is learning.

**Comment by KiLL4FuNK** [comment]
@sol666

remember to only use ublock no adblock or other extensions

It works well for me

**Comment by sol666** [comment]
I did. The website reacted to it and barred my access. It seems to know all
the adblocks. And those that work only do so the first time they are
installed. It learns them by second time the website is accessed

**Comment by KiLL4FuNK** [comment]
shows a capture of the problem?
